consensus/src/proposal/step.rs

```diff
@@ -63,12 +64,18 @@ impl<T: Operations + 'static, D: Database> ProposalStep<T, D> {
             let failed_certificates =
                 ctx.sv_registry.lock().await.get_failed_certs(iteration);
 
+            let generation_timout = ctx
+                .iter_ctx
+                .get_timeout(node_data::StepName::Proposal)
+                .saturating_sub(Duration::from_secs(1));
```

Comment:
Should the choice of 1 second less be configurable maybe?

Reply:
Agree, probably it would be a nice change to include it into the rusk.toml

Do you mind if I open an issue to be tracked for the `master` branch? This is for ITN and (at least mine) goal is to fast iterate on top of that

Reply:
Nevermind, I've added the configuration in this branch